susemanager-utils/susemanager-sls/salt/certs/init.sls

```diff
@@ -19,4 +19,10 @@
 {% else -%}
 {{ includesls(grains['os'], grains['osrelease_info']|first|string) }}
 {% endif -%}
+{% elif grains['os_family'] == 'Debian' %}
+{% if grains['os'] == 'Ubuntu' %}
```

Comment:
Maybe this condition can be removed. I think this is generic enough to work on both stock Debian and Ubuntu.

Reply:
makes sense, it's now the same for Debian and Ubuntu